applications/nrf5340_audio/src/bluetooth/ble_audio_services.c

```diff
@@ -117,6 +127,156 @@ static void vcs_flags_cb_handler(struct bt_vcs *vcs, int err, uint8_t flags)
 	}
 }
 
+#if (CONFIG_AUDIO_DEV == GATEWAY)
+#if (CONFIG_BT_VCS_CLIENT)
```

Comment:
Would it be enough to check for VCS_CLIENT?

Reply:
Agree, done